pkg/cluster/cache_whitebox_test.go

```diff
@@ -392,19 +336,19 @@ func TestRefreshCache(t *testing.T) {
 
 func TestMultipleActionsInParallel(t *testing.T) {
 	// given
+	clusterForTest := newTestCachedToolchainCluster(t, "clusterForTest", ready)
+
 	defer resetClusterCache()
 	var latch sync.WaitGroup
 	latch.Add(1)
 	var waitForFinished sync.WaitGroup
 
-	memberCluster := newTestCachedToolchainCluster(t, "memberCluster", Member, ready)
-	hostCluster := newTestCachedToolchainCluster(t, "hostCluster", Host, ready)
 	clusterCache.refreshCache = func() {
-		clusterCache.addCachedToolchainCluster(memberCluster)
-		clusterCache.addCachedToolchainCluster(hostCluster)
+		clusterCache.addCachedToolchainCluster(clusterForTest)
+
 	}
 
-	for _, clusterToTest := range []*CachedToolchainCluster{memberCluster, hostCluster} {
+	for _, clusterToTest := range []*CachedToolchainCluster{clusterForTest} {
```

Comment:
I think we can drop this for loop right ? If it contains just one element in list.